UnityProject/Assets/Scripts/Machines/Jukebox.cs

```diff
@@ -274,12 +290,12 @@ private void OnDamageReceived(DamageInfo damageInfo)
 		}
 	}
 
-	private void UpdateGUI()
+	private async void UpdateGUIAsync()
 	{
 		List<ElementValue> valuesToSend = new List<ElementValue>();
 		valuesToSend.Add(new ElementValue() { Id = "TextTrack", Value = Encoding.UTF8.GetBytes(TrackPosition) });
-		valuesToSend.Add(new ElementValue() { Id = "TextSong", Value = Encoding.UTF8.GetBytes(SongName) });
-		valuesToSend.Add(new ElementValue() { Id = "TextArtist", Value = Encoding.UTF8.GetBytes(Artist) });
+		valuesToSend.Add(new ElementValue() { Id = "TextSong", Value = Encoding.UTF8.GetBytes(await GetSongNameAsync()) });
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)